astropy/visualization/wcsaxes/coordinate_helpers.py

```diff
@@ -427,7 +427,13 @@ def set_ticks_visible(self, visible):
         self.ticks.set_visible(visible)
 
     def set_ticklabel(
-        self, color=None, size=None, pad=None, exclude_overlapping=None, **kwargs
+        self,
+        color=None,
+        size=None,
+        pad=None,
+        exclude_overlapping=None,
+        simplify=True,
```

Comment:
I'd recommend making any new optional argument (beyond second or third position) keyword-only:
- prevents users from writing obscure boolean flag positionally (however unlikely)
- makes deprecation much easier if it's ever needed

```suggestion
        *,
        simplify=True,
```

Reply:
Heh, I totally agree, and the only reason I didn't do that was because I think most of WCSAxes code pre-dates that best practice, and I thought it might be a bit weird to suddenly specify this particular argument to be keyword-only